include/deal.II/base/parameter_acceptor.h

```diff
@@ -736,6 +760,14 @@ ParameterAcceptorProxy<SourceClass>::parse_parameters(ParameterHandler &prm)
   SourceClass::parse_parameters(prm);
 }
 
+
+
+inline unsigned int
+ParameterAcceptor::get_acceptor_id() const
+{
+  return acceptor_id;
+}
```

Comment:
The only place where you call this is in the `.cc` file -- might as well move the function there.